docs/topics/power-assert.md

```diff
@@ -0,0 +1,400 @@
+[//]: # (title: Power-assert compiler plugin)
+
+> The Power-assert compiler plugin is [Experimental](components-stability.md).
+> It may be changed at any time. Use it only for evaluation purposes.
+> We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
+>
+{type="warning"}
+
+The Kotlin Power-assert compiler plugin improves the debugging experience
+by providing detailed failure messages with contextual information.
+It simplifies the process of writing tests by automatically generating 
+intermediate values in failure messages. 
+It helps you understand why a test failed without needing complex assertion libraries.
+
+The Power-assert plugin key features:
+
+* **Enhanced error messages**: The plugin captures and displays the values of variables and 
+   sub-expressions within the assertion to clearly identify the cause of failure.
+* **Simplified testing**: Automatically generates informative failure messages,
+   reducing the need for complex assertion libraries.
+* **Supports multiple functions**: By default, it transforms `assert()` function calls but can also transform other functions,
+   such as `require()`, `check()`, and `assertTrue()`.
+
+## Apply the plugin
+
+To enable the Power-assert plugin, configure your `build.gradle(.kts)` file as follows:
+
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+// build.gradle.kts
+plugins {
+    kotlin("multiplatform") version "2.0.0"
+    kotlin("plugin.power-assert") version "2.0.0"
+}
+```
+
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
+// build.gradle
+plugins {
+    id 'org.jetbrains.kotlin.multiplatform' version '2.0.0'
+    id 'org.jetbrains.kotlin.plugin.power-assert' version '2.0.0'
+}
+```
+
+</tab>
+</tabs>
+
+## Configure the plugin
+
+The Power-assert plugin provides several options to customize its behavior:
+
+* **`functions`**: A list of fully-qualified function paths. The Power-assert plugin will transform the calls to these functions. If empty, only `kotlin.assert()` calls will be transformed by default.
+* **`includedSourceSets`**: A list of Gradle source sets that the Power-assert plugin will transform. If empty, all _test source sets_ will be transformed by default.
```

Comment:
"If empty" here reads to me like "if an empty list is provided", but what I think is meant is "if the option is not specified" (because that is the default). Wdyt?